src/imageTools/magnify2.js

```diff
@@ -108,9 +109,9 @@ function drawMagnificationTool (eventData) {
   // Calculate the on-canvas location of the mouse pointer / touch
   const canvasLocation = external.cornerstone.pixelToCanvas(eventData.element, eventData.currentPoints.image);
 
-  if (eventData.isTouchEvent === true) {
-    canvasLocation.y -= 1.25 * getSize;
-  }
+  /* If (eventData.isTouchEvent === true) {
+    canvasLocation.y -= 20;
+  }*/
```

Comment:
I agree that an offset of `1.25 * getSize` is not a good approximation. Should be good to remove this comment as you're modifying styling for `magnify.style.top` down below.